### PR TITLE
Update ServiceRequest guidance (FHIR-47007) and examples ( FHIR-47147)

### PR DESCRIPTION
**Pull request by johnscarter**
[FHIR-47007](https://jira.hl7.org/browse/FHIR-47007) Added guidance on the use of the insurance element. After discussion with MW have added the text to the insurance.comment element as well as the profile-specific implementation guidance section.

[FHIR-47147](https://jira.hl7.org/browse/FHIR-47147) Updated path and imag order examples to use the scoped HPI-O identifier system

### CODE REVIEW

File: fsh-generated/resources/ActorDefinition-au-erequesting-actor-patient.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "title": "AU eRequesting Patient",
```
Comment (dbojicic) [inline fsh-generated/resources/ActorDefinition-au-erequesting-actor-patient.json:6]: Commenting here to get an inline comment. 
fsh-generated files are not tracked anymore. Please remove from this PR.

File: input/pagecontent/changes.md [hunk 2]
```diff
@@ -14,4 +14,7 @@ This change log documents the significant updates and resolutions implemented fr
   - made ServiceRequest.encounter mandatory (1..1) [FHIR-47008](https://jira.hl7.org/browse/FHIR-47008)
   - added Must Support to ServiceRequest.priority [FHIR-46939](https://jira.hl7.org/browse/FHIR-46939)
   - AU eRequesting ServiceRequest profile renamed to be AU eRequesting Diagnostic Request [FHIR-46842](https://jira.hl7.org/browse/FHIR-46842)
+  - updated profile-specific implementation guidance for `insurance` element ([FHIR-47007](https://jira.hl7.org/browse/FHIR-47007))
+  - updated identifier system in order examples ([FHIR-47147](https://jira.hl7.org/browse/FHIR-47147))
```
Comment (dbojicic) [inline input/pagecontent/changes.md:18]: Please remove from the PR, we only track substantive changes in the change log.

**Review by dbojicic — CHANGES_REQUESTED** [review]
Just some minor changes, please see comments.

File: input/pagecontent/changes.md (hunk 2 above)
Comment (johnscarter) [reply input/pagecontent/changes.md]: done

File: fsh-generated/resources/ActorDefinition-au-erequesting-actor-patient.json (hunk 1 above)
Comment (dbojicic) [reply fsh-generated/resources/ActorDefinition-au-erequesting-actor-patient.json]: I can see the deleted file but fsh-generated and the deleted file are still showing in this PR which makes me think that fsh-generated wasn't properly untracked. If it was, fsh-generated and files inside it, whether they are new, modified or deleted, should not be shown in the PR.

**Review by dbojicic — CHANGES_REQUESTED** [review]
One minor comment.

File: fsh-generated/resources/ActorDefinition-au-erequesting-actor-patient.json (hunk 1 above)
Comment (dbojicic) [reply fsh-generated/resources/ActorDefinition-au-erequesting-actor-patient.json]: Still showing in PR.

**Review by dbojicic — CHANGES_REQUESTED** [review]
One comment to be addressed.

File: fsh-generated/resources/ActorDefinition-au-erequesting-actor-patient.json (hunk 1 above)
Comment (johnscarter) [reply fsh-generated/resources/ActorDefinition-au-erequesting-actor-patient.json]: done

**Review by dbojicic — APPROVED** [review]

**Review by michael-wilson-au — APPROVED** [review]

**Pull request merged**